awscli/customizations/cloudformation/artifact_exporter.py

```diff
@@ -417,6 +416,15 @@ class ServerlessRepoApplicationLicense(Resource):
     PACKAGE_NULL_PROPERTY = False
 
 
+class StepFunctionsStateMachineDefinitionResource(ResourceWithS3UrlDict):
+    RESOURCE_TYPE = "AWS::StepFunctions::StateMachine"
+    PROPERTY_NAME = "DefinitionS3Location"
+    BUCKET_NAME_PROPERTY = "Bucket"
+    OBJECT_KEY_PROPERTY = "Key"
+    VERSION_PROPERTY = "Version"
+    PACKAGE_NULL_PROPERTY = True
```

Comment:
Should the `PACKAGE_NULL_PROPERTY` be set to `False` here? Specifically, it matches what the ApiGateway resource does: https://github.com/aws/aws-cli/blob/2998af688919ea8c770a225067dafcc8e6ab4d78/awscli/customizations/cloudformation/artifact_exporter.py#L378. Also based on the issue where it got added: https://github.com/aws/aws-cli/pull/2322, it seems needed as users could define the step function definitions inline in the template so we would not necessarily always want to be uploading to S3 and setting the S3 property.

Reply:
Yes, it should be `False`, thanks for pointing it out.